cmd/vela-terraform/main.go

```diff
@@ -373,6 +373,7 @@ func run(c *cli.Context) error {
 			Parallelism:      c.Int("parallelism"),
 			Refresh:          c.Bool("refresh"),
 			State:            c.String("state"),
+			Out:              c.String("state_out"),
```

Comment:
Suggestion to rename the `Out` struct field to `StateOut`? Seems like that would follow the pattern for all the other fields where the flag name has a `_` separator.

Reply:
We can. It was originally different because the terraform CLI has it different,
https://www.terraform.io/docs/cli/commands/plan.html

I'm open to either leaving it, taking your suggestion, or even changing the `state_out` param for plan to be `out`

Reply:
Makes sense. I think we're good.